engi.go

```diff
@@ -25,32 +21,53 @@ var (
 	scaleOnResize   = false
 	fpsLimit        = 120
 	headless        = false
+	vsync           = true
```

Comment:
VSync refers to the refresh-rate of the monitor. A game can update the graphics (for example) 120 times per second. But if the monitor only refreshes 60 times per second, half of those graphics-updates aren't that useful - most of the time, you won't see them anyway. 

VSync tells the GPU to "synchronize" with the monitor: make sure we're only doing GPU rendering when the monitor is refreshing. This usually means, in practice, the FPS-limit becomes `max(user_defined_fpsLimit, monitor_refresh_rate)`, in pseudo-code.